alpine-apache/root/etc/services.d/apache/finish

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/execlineb -S0
+
+# only tell s6 to bring down the entire container, if it isn't already doing so
+# http://skarnet.org/software/s6/s6-supervise.html
+if -n { s6-test $# -ne 0 }
```

Comment:
This code has since been changed. You can see the updated code in https://github.com/smebberson/docker-alpine/blob/master/alpine-nginx/root/etc/services.d/nginx/finish. Would you mind updating accordingly? I am going to be bringing those changes to `alpine-consul-nginx` (etc).